src/main/java/com/kodar/academy/Library/model/dto/book/BookCreateDTO.java

```diff
@@ -0,0 +1,74 @@
+package com.kodar.academy.Library.model.dto.book;
+
+import com.kodar.academy.Library.model.dto.author.AuthorDTO;
+import com.kodar.academy.Library.model.dto.genre.GenreDTO;
+
+import java.time.LocalDateTime;
+import java.util.Set;
+
+public class BookCreateDTO {
```

Comment:
Request objects received with HTTP request should be different classes than those returned. As an idea, you could name received object BookCreateRequest and returned BookCreateDTO